pkg/config/nerdctl_config_applier.go

```diff
@@ -91,8 +91,8 @@ func addLineToBashrc(fs afero.Fs, profileFilePath string, profStr string, cmd st
 func updateEnvironment(fs afero.Fs, fc *Finch, finchDir, homeDir, limaVMHomeDir string) error {
 	cmdArr := []string{
 		`export DOCKER_CONFIG="$FINCH_DIR"`,
-		"[ -L /usr/local/bin/docker-credential-ecr-login ] " +
-			`|| sudo ln -s "$FINCH_DIR"/cred-helpers/docker-credential-ecr-login /usr/local/bin/`,
+		`for f in "$FINCH_DIR"/cred-helpers/docker-credential-*; do ` +
+			`[ -L "/usr/local/bin/$(basename ${f})" ] || sudo ln -s "${f}" "/usr/local/bin/$(basename ${f})"; done`,
```

Comment:
The logic seems to be only checking for if a symbolic link exists. If there is a regular file by the same name, this may still fail.